src/bench/bench.cpp

```diff
@@ -41,18 +42,47 @@ void GenerateTemplateResults(const std::vector<ankerl::nanobench::Result>& bench
 
 } // namespace
 
-benchmark::BenchRunner::BenchmarkMap& benchmark::BenchRunner::benchmarks()
+namespace benchmark {
+
+std::map<std::string, uint8_t> map_priority_level = {
```

Comment:
nit: I think `priority` better highlights the difference between the aggregated priority and the individual priority level
```suggestion
std::map<std::string, uint8_t> map_label_priority = {
```